tslint-server/src/server.ts

```diff
@@ -109,12 +109,15 @@ folder using \'npm install tslint\' or \'npm install -g tslint\' and then press
 // Options passed to tslint
 let options: tslint.ILinterOptions = {
 	formatter: "json",
-	configuration: {},
+	fix: false,
```

Comment:
@egamma yeah I guess this is just defaults then it overwrites what is passed in. Thanks for clarifying.